reactor-netty-http/src/main/java/reactor/netty/http/server/NonSslRedirectHandler.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2011-Present VMware, Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.netty.http.server;
+
+import io.netty.channel.ChannelFutureListener;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelInboundHandlerAdapter;
+import io.netty.handler.codec.http.DefaultFullHttpResponse;
+import io.netty.handler.codec.http.HttpRequest;
+import io.netty.handler.codec.http.HttpResponse;
+import io.netty.handler.codec.http.HttpResponseStatus;
+
+import javax.annotation.Nullable;
+
+import static io.netty.handler.codec.http.HttpHeaderNames.HOST;
+import static io.netty.handler.codec.http.HttpHeaderNames.LOCATION;
+
+/**
+ * @author James Chen
+ */
+final class NonSslRedirectHandler extends ChannelInboundHandlerAdapter {
+
+	private static final String HTTP_PROTOCOL = "http://";
+	private static final String HTTPS_PROTOCOL = "https://";
+
+	@Override
+	public void channelRead(ChannelHandlerContext ctx, Object msg) {
```

Comment:
```
> Task :reactor-netty-http:compileJava
.../reactor-netty/reactor-netty-http/src/main/java/reactor/netty/http/server/NonSslRedirectHandler.java:46: warning: [FutureReturnValueIgnored] Return value of methods returning Future must be checked. Ignoring returned Futures suppresses exceptions thrown from the code that completes the Future.
                                ctx.close();
                                         ^
    (see https://errorprone.info/bugpattern/FutureReturnValueIgnored)
  Did you mean to remove this line?
1 warning
```

Let's fix this warning:

```suggestion
	@SuppressWarnings("FutureReturnValueIgnored")
	public void channelRead(ChannelHandlerContext ctx, Object msg) {
```

Reply:
Done.